src/components/Layout/Page.tsx

```diff
@@ -35,30 +40,26 @@ function PageInset(props: { children: React.ReactNode }) {
   )
 }
 
-interface SectionProps {
-  children: React.ReactNode
+interface SectionProps extends BoxProps {
+  inheritBackground?: boolean
   backgroundColor?: React.CSSProperties["backgroundColor"]
   bottom?: boolean
   brandColored?: boolean
-  grow?: number
   noPadding?: boolean
-  shrink?: number
   pageInset?: boolean
   top?: boolean
-  style?: React.CSSProperties
 }
 
-// tslint:disable-next-line no-shadowed-variable
 const Section = React.memo(function Section(props: SectionProps) {
-  const background = props.brandColored ? primaryBackground : props.backgroundColor || "#fcfcfc"
+  const background = props.brandColored
+    ? primaryBackground
+    : props.inheritBackground
+    ? undefined
+    : props.backgroundColor || "#fcfcfc"
   const isSmallScreen = useIsMobile()
 
-  const className = [
```

Comment:
Why did you remove the `className` with the iOS notch classes? This looks like it's gonna break the layout on iOS now.

Reply:
I tested this on different virtual iOS devices (iPhone X 11.0.1, iPhone 11 Pro Max 13.2.2, iPhone 8 13.2.2, iPhone 7 11.0.1) and it looks just fine.
So I'd assume it is safe to remove it as I could not find any scenario where it seemed to change anything. 
But if you don't feel good or confident about removing it, we can keep it anyways.

Reply:
Cool. Then let's try it.